plugwise/constants.py

```diff
@@ -539,15 +538,15 @@
     },
     "smile_thermo_v4": {
         "type": "thermostat",
-        "friendly_name": "Anna",
+        "friendly_name": "Smile",
```

Comment:
Looking back at these, we could summarize them to `if 'smile_thermo' in ...` and then if v1, set legacy. But thats refinement.

Reply:
Nice! Some time ago, I was looking to get rid of the legacy-part of the dict :)

Reply:
Implemented.